# Nobel prizes

<H1> Executive Summary </H1>

we've loaded the data sucessfully of Nobel prize,from 1901 t0 2016, however the data is not complete about 11% of it is missing -(refere to 1. of conclusion), the data has 911 entries , made out of 18 colums.

Through the data we've been able to draw three main conclusions.

one: The Nobel  Prize is,highly sexiest. your are 18 times more likely to win the price if you are male(only 1 out of 18 winners are female) -(refer to .2 in conclusion), 

two: the Nobel Prize is was amd still is Eurocentric,9 out of the top ten  wining countries , are  werstern countries.they  make up  about 61% of the total prizes won so far.-(refer to .3 in conclusion),

three: THE AGE?

## 1. Loading the data
<p><img style="float: right;margin:5px 20px 5px 1px; max-width:250px" src="https://s3.amazonaws.com/assets.datacamp.com/production/project_441/img/Nobel_Prize.png"></p>
<p>The Nobel Prize is perhaps the world's most well known scientific award. Except for the honor, prestige and substantial prize money the recipient also gets a gold medal showing Alfred Nobel (1833 - 1896) who established the prize. Every year it's given to scientists and scholars in the categories chemistry, literature, physics, physiology or medicine, economics, and peace. The first Nobel Prize was handed out in 1901, and at that time the Prize was very Eurocentric and male-focused, but nowadays it's not biased in any way whatsoever. Surely. Right?</p>
<p>Well, we're going to find out! The Nobel Foundation has made a dataset available of all prize winners from the start of the prize, in 1901, to 2016. Let's load it in and take a look.</p>

In [55]:
#pt1
# Loading in required libraries: you will need pandas, seaborn and numpy
# ... YOUR CODE FOR TASK 1 ...
import pandas as pd
import seaborn as sb
import numpy as np

#pt2
# Reading in the Nobel Prize data
nobel = pd.read_csv (r"C:\Users\bongekile\Desktop\A Visual History of Nobel Prize Winners\datasets\nobel.csv")
display(nobel.head(6))
#pt3
# Explore the data - are there missing values? How many entries do we have?
# ... YOUR CODE FOR TASK 1 ...


space = nobel.isnull().sum() 
print("we have spaces.all of the True's represent colombs with spaces" )

print(space.sum())
print("Total entries")
print (len(nobel))
print (nobel.shape)


year    category                                           prize  \
0  1901   Chemistry               The Nobel Prize in Chemistry 1901   
1  1901  Literature              The Nobel Prize in Literature 1901   
2  1901    Medicine  The Nobel Prize in Physiology or Medicine 1901   
3  1901       Peace                      The Nobel Peace Prize 1901   
4  1901       Peace                      The Nobel Peace Prize 1901   
5  1901     Physics                 The Nobel Prize in Physics 1901   

                                          motivation prize_share  laureate_id  \
0  "in recognition of the extraordinary services ...         1/1          160   
1  "in special recognition of his poetic composit...         1/1          569   
2  "for his work on serum therapy, especially its...         1/1          293   
3                                                NaN         1/2          462   
4                                                NaN         1/2          463   
5  "in recognition of the extraordinary services ...         1/1            1   

  laureate_type                     full_name  birth_date          birth_city  \
0    Individual  Jacobus Henricus van 't Hoff  1852-08-30           Rotterdam   
1    Individual               Sully Prudhomme  1839-03-16               Paris   
2    Individual        Emil Adolf von Behring  1854-03-15   Hansdorf (Lawice)   
3    Individual             Jean Henry Dunant  1828-05-08              Geneva   
4    Individual                Frédéric Passy  1822-05-20               Paris   
5    Individual        Wilhelm Conrad Röntgen  1845-03-27  Lennep (Remscheid)   

       birth_country   sex   organization_name organization_city  \
0        Netherlands  Male   Berlin University            Berlin   
1             France  Male                 NaN               NaN   
2   Prussia (Poland)  Male  Marburg University           Marburg   
3        Switzerland  Male                 NaN               NaN   
4             France  Male                 NaN               NaN   
5  Prussia (Germany)  Male   Munich University            Munich   

  organization_country  death_date death_city death_country  
0              Germany  1911-03-01     Berlin       Germany  
1                  NaN  1907-09-07   Châtenay        France  
2              Germany  1917-03-31    Marburg       Germany  
3                  NaN  1910-10-30     Heiden   Switzerland  
4                  NaN  1912-06-12      Paris        France  
5              Germany  1923-02-10     Munich       Germany

we have spaces.all of the True's represent colombs with spaces
1912
Total entries
911
(911, 18)


## 2. So, who gets the Nobel Prize?
<p>Just looking at the first couple of prize winners, or Nobel laureates as they are also called, we already see a celebrity: Wilhelm Conrad Röntgen, the guy who discovered X-rays. And actually, we see that all of the winners in 1901 were guys that came from Europe. But that was back in 1901, looking at all winners in the dataset, from 1901 to 2016, which sex and which country is the most commonly represented? </p>
<p>(For <em>country</em>, we will use the <code>birth_country</code> of the winner, as the <code>organization_country</code> is <code>NaN</code> for all shared Nobel Prizes.)</p>

In [57]:
# Display the number of (possibly shared) Nobel Prizes handed
# out between 1901 and 2016
# ... YOUR CODE FOR TASK 2 
print("the posible number of shared Nobel prices")
share = nobel["organization_country"].isnull().sum()
print(share)



# Display the number of prizes won by male and female recipients.
# ... YOUR CODE FOR TASK 2 ...
print("number Nobel Prices won by females")
fem =nobel['sex'] == "Female"
bubu =fem.sum()
print(bubu)


print("number of Nobel prices won by males")
mal = nobel['sex'] == "Male"
baba = mal.sum()
print(baba)

print("number of Nobel prices won by both male and females")
zonke = mal | fem
malfe = zonke.sum()
print(malfe)
# Display the number of prizes won by the top 10 nationalities.
# ... YOUR CODE FOR TASK 2 ...
top10 = nobel['birth_country'].value_counts().head(10)
print("These are the top ten countries")
print(top10)
print("put together they have colectivly won")
print (top10.sum())

the posible number of shared Nobel prices
244
number Nobel Prices won by females
49
number of Nobel prices won by males
836
number of Nobel prices won by both male and females
885
These are the top ten countries
United States of America    259
United Kingdom               85
Germany                      61
France                       51
Sweden                       29
Japan                        24
Netherlands                  18
Canada                       18
Italy                        17
Russia                       17
Name: birth_country, dtype: int64
put together they have colectivly won
579


about 26 people   without a sex, total entries - total male & female  = 911 - 885 = 26

## 3. USA dominance
<p>Not so surprising perhaps: the most common Nobel laureate between 1901 and 2016 was a man born in the United States of America. But in 1901 all the winners were European. When did the USA start to dominate the Nobel Prize charts?</p>
    

In [48]:
# Calculating the proportion of USA born winners per decade
nobel['usa_born_winner'] =  nobel['birth_country'] == 'United States of America'
year = pd.Series(nobel['year'])
nobel ['decade'] = (np.floor(year / 10) * 10).astype(int)
prop_usa_winners = nobel.groupby('decade', as_index=False)['usa_born_winner'].mean()

# Display the proportions of USA born winners per decade
# ... YOUR CODE FOR TASK 3 ...
display(prop_usa_winners)



decade  usa_born_winner
0     1900         0.017544
1     1910         0.075000
2     1920         0.074074
3     1930         0.250000
4     1940         0.302326
5     1950         0.291667
6     1960         0.265823
7     1970         0.317308
8     1980         0.319588
9     1990         0.403846
10    2000         0.422764
11    2010         0.292683

## 4. USA dominance, visualized
<p>A table is OK, but to <em>see</em> when the USA started to dominate the Nobel charts we need a plot!</p>

In [14]:
#using seaborn, plot the proportion of US winners by decade

What can you conclude from the graph?

## 5. What is the gender of a typical Nobel Prize winner?
<p>So the USA became the dominating winner of the Nobel Prize first in the 1930s and had kept the leading position ever since. But one group that was in the lead from the start, and never seems to let go, are <em>men</em>. Maybe it shouldn't come as a shock that there is some imbalance between how many male and female prize winners there are, but how significant is this imbalance? And is it better or worse within specific prize categories like physics, medicine, literature, etc.?</p>

In [60]:
# Calculating the proportion of female laureates per decade
nobel['female_winners'] = nobel["sex"] == "Female"

year = pd.Series(nobel['year'])
nobel ['decade'] = (np.floor(year / 10) * 10).astype(int)
prop_female_winners = nobel.groupby('decade', as_index=False)['female_winners'].mean()

# Plot proportion of female winners by decade, grouped by prize category (physics, medicine, etc)
# ... YOUR CODE FOR TASK 5 ...
display(prop_female_winners)

decade  female_winners
0     1900        0.052632
1     1910        0.025000
2     1920        0.037037
3     1930        0.053571
4     1940        0.069767
5     1950        0.000000
6     1960        0.037975
7     1970        0.038462
8     1980        0.041237
9     1990        0.067308
10    2000        0.089431
11    2010        0.097561



```
# This is formatted as code
 sb.lineplot(prop_female_winners["decade"],prop_female_winners["female_winners"])
plt.show()

```
this id formeted as code diplay your graph here




## 6. The first woman to win the Nobel Prize
<p>The plot above is a bit messy as the lines are overplotting. But it does show some interesting trends and patterns. Overall the imbalance is pretty large with physics, economics, and chemistry having the largest imbalance. Medicine has a somewhat positive trend, and since the 1990s the literature prize is also now more balanced. The big outlier is the peace prize during the 2010s, but keep in mind that this just covers the years 2010 to 2016.</p>
<p>Given this imbalance, who was the first woman to receive a Nobel Prize? And in what category?</p>


In [0]:
# Picking out the first woman to win a Nobel Prize
# ... YOUR CODE FOR TASK 6 ...

## 7. Repeat laureates
<p>For most scientists/writers/activists a Nobel Prize would be the crowning achievement of a long career. But for some people, one is just not enough, and few have gotten it more than once. Who are these lucky few? (Having won no Nobel Prize myself, I'll assume it's just about luck.)</p>

In [0]:
# Selecting the laureates that have received 2 or more prizes.
# ... YOUR CODE FOR TASK 7 ...



How many people have received 2 or more prizes?

## 8. How old are you when you get the prize?
<p>The list of repeat winners contains some illustrious names! We again meet Marie Curie, who got the prize in physics for discovering radiation and in chemistry for isolating radium and polonium. John Bardeen got it twice in physics for transistors and superconductivity, Frederick Sanger got it twice in chemistry, and Linus Carl Pauling got it first in chemistry and later in peace for his work in promoting nuclear disarmament. We also learn that organizations also get the prize as both the Red Cross and the UNHCR have gotten it twice.</p>
<p>But how old are you generally when you get the prize?</p>

In [44]:
# Converting birth_date from String to datetimeS
nobel['birth_date'] = ...

# Calculating the age of Nobel Prize winners
nobel['age'] = ...

# What is the mean age of winners and it's standard devation? What is the median age?

## 9. Age differences between prize categories


In [0]:
#Plot the age of nobel candidates by decade
#YOUR CODE FOR TASK 9 ...

# Plot the age of nobel candidates by decade and prize category

What can you conclude about the age at which people win different types of Nobel Prizes?

## 10. Oldest and youngest winners
<p>In the peace category there is a winner around 2010 that seems exceptionally young. This begs the questions, who are the oldest and youngest people ever to have won a Nobel Prize?</p>

## Congratulations! You've reached the end of the challenge!

<H1> final remarks </H1>

further , we can look at is the corolation with a countries power , and power versers , its number of winners , the top three countires twe who win , hapen to be the most powerfull countries in history.
we can also corelate with when each countries when wining the most , which speaks to were they at the hight of their power etc.

we can ofcourse do the same for sex, distribution of power through gender. and through age.this way we can indiractly track how things are changing or not , in terms of eurocentrism and gender equality and the propagation of the youth


<h1> conclusion</h1>

1.how we got to 11% of the data is mising : total mising feilds/ number of entries * number of rows *100 = 1912 / 911 * 18 * 100 = 11%.

2 how we got to 18 times mor likely to win as a man:  total male & female winners / total female winners = 885 / 49 this way = 18,06.

3 how we fot 61% : top ten (total prices - japan / total entries ) * 100 = (579 -24 / 911) * 100 = 60.92%.